src/helpers/helper.js

```diff
@@ -1,5 +1,10 @@
+import faker from 'faker';
 import { TOP } from '../constants/constants';
 
+export const generateId = faker.random.uuid;
+
+export const validateInput = (input, regex) => (input ? regex.test(input) : true);
```

Comment:
you won't need anything but regex.test(input) if you add minLength attribute to each input field